nsis/src/nsis/format/NsisScriptHeader.java

```diff
@@ -11,35 +11,40 @@
 import nsis.file.NsisConstants;
 
 public class NsisScriptHeader implements StructConverter {
+	public final int flags;
+	private byte[] siginfo;
 	private byte[] magic;
 	public final int inflatedHeaderSize;
 	public final int archiveSize;
 	public final int compressedHeaderSize;
-	public final int flags;
 	private final static Structure STRUCTURE;
 
 	static {
 		STRUCTURE = new StructureDataType("script_header", 0);
-		STRUCTURE.add(STRING, NsisConstants.NSIS_MAGIC.length, "magic", null);
-		STRUCTURE.add(DWORD, DWORD.getLength(), "inflated_header_size", null);
-		STRUCTURE.add(DWORD, DWORD.getLength(), "header_size", null);
-		STRUCTURE.add(DWORD, DWORD.getLength(), "compressed_header_size", null);
-		STRUCTURE.add(DWORD, DWORD.getLength(), "flags", null);
+		STRUCTURE.add(DWORD, DWORD.getLength(), "flags", "First header flags (FH_FLAGS_*)");
+		STRUCTURE.add(STRING, NsisConstants.NSIS_SIGINFO.length, "siginfo", "0xdeadbeef (FH_SIG)");
+		STRUCTURE.add(STRING, NsisConstants.NSIS_MAGIC.length, "nsinst",
```

Comment:
it isn't clear to me what "nsinst" means. Should it be left as "magic" or "nsis_magic"?

Reply:
I used the names declared in the NSIS spec so it matches: https://sourceforge.net/p/nsis/code/HEAD/tree/NSIS/trunk/Source/exehead/fileform.h#l239

That argument of each add appears as a variable name in Ghidra and it is followed by a comment, so I think it's a good idea that they match the original name in the NSIS spec
![image](https://user-images.githubusercontent.com/24403370/85337617-d0e1f580-b4ae-11ea-979d-8eefe3df0bae.png)

Reply:
*Add a comment to the NSIS spec